db/migrate/20240226193757_migrate_templates_to_mappings.rb

```diff
@@ -0,0 +1,5 @@
+class MigrateTemplatesToMappings < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
Let's use `up` here and leave `down` empty (unless we can recreate the `.template` files). If a dev somehow decides to rollback, the migration service will get called again.

```suggestion
  def up
```